### PR TITLE
fix(tekton): Use a working and useful versions directory

### PR DESCRIPTION
**Pull request by abayer**
#### Submitter checklist

- [x] Change is code complete and matches issue description.
- [x] Change is covered by existing or new tests.

#### Description

We *were* using the typoed `cmmon_versions`, but once that typo was
fixed, we weren't using anything. Changing to just use the
now-existant `common_versions` wasn't particularly desirable, since it
messed with expected volumes etc, so instead let's use
`stable_versions`, which matches up with the pod templates we define
for the test.

#### Special notes for the reviewer(s)

/assign @hferentschik 
/assign @dwnusbaum 

#### Which issue this PR fixes

n/a

### CODE REVIEW

File: pkg/jx/cmd/step_create_task_test.go [hunk 1]
```diff
@@ -41,11 +41,11 @@ func TestGenerateTektonCRDs(t *testing.T) {
 	log.SetOutput(ioutil.Discard)
 	defer log.SetOutput(os.Stderr)
 
-	testVersionsDir := path.Join("test_data", "cmmon_versions")
```
Comment (hferentschik) [inline pkg/jx/cmd/step_create_task_test.go]: So what happens here with the stuff under common_versions?

**Comment by abayer** [comment]
Rebased!

`common_versions` is still being used by `version_test`, so I left it as is.

**Pull request merged**